lib/msal-browser/src/network/FetchClient.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { INetworkModule } from "msal-common";
+
+export class FetchClient implements INetworkModule {
+
+    /**
+     * XHR client for JSON endpoints
+     * https://www.npmjs.com/package/async-promise
+     * @param url 
+     * @param requestParams 
+     * @param enableCaching 
+     */
+    async sendRequestAsync(url: string, requestParams: RequestInit, enableCaching?: boolean): Promise<any> {
```

Comment:
1. This signature feels too tied to fetch. I suppose it would be fine if we decided to use `node-fetch` for Node, for example, but I still think we should consider designing our own API, instead of just using fetch's as-is.
2. Don't think `enableCaching` needs to be an option anymore.